templates/procuration/landing.html.twig

```diff
@@ -30,7 +30,7 @@
                         <div class="b__nudge--bottom text--summary">
                             Je veux voter au nom d'un(e) citoyen(ne) de ma ville.
                         </div>
-                        <a href="{{ path('app_procuration_proxy_proposal') }}" class="btn btn--primary">
+                        <a href="{{ path('app_procuration_choose_election', { action: 'proposition' }) }}" class="btn btn--primary">
```

Comment:
I think you can use the `ACTION_PROPOSAL` const here ?

Reply:
Fixed in https://github.com/EnMarche/en-marche.fr/pull/1948/commits/81f9332262da308744868890e8828cf30cf4fce4.